app/connectors/SaConnector.scala

```diff
@@ -51,6 +52,7 @@ class SaConnector @Inject()(appConfig: FrontendAppConfig, http: HttpClient) exte
   def getBusinessDetails(value: String, identifier: String)
                         (implicit hc: HeaderCarrier, ec: ExecutionContext, request: ServiceInfoRequest[AnyContent]): Future[Option[BusinessDetails]] = {
     http.GET[BusinessDetails](serviceBusinessDetails(value, identifier)).map { result =>
+      infoLog(s"[SaConnector][getBusinessDetails] attempted with: [value: {$value}, identifier: {$identifier}]")
```

Comment:
This will log out a user's nino. I don't think we should be logging tax identifiers